mentorship ios/ViewModels/RequestActionViewModel.swift

```diff
@@ -0,0 +1,11 @@
+//
+//  RequestActionViewModel.swift
+//  Created on 18/07/20
+//  Created for AnitaB.org Mentorship-iOS 
+//
+
+import Foundation
+
+class RequestActionViewModel: ObservableObject {
+    let requestActionAPI = RequestActionAPI()
```

Comment:
Your service should be in View not in ViewModel.

Reply:
Okay